src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_ListViewBase.cs

```diff
@@ -4591,6 +4591,59 @@ public async Task When_ScrollIntoView_Containers_With_Varying_Heights()
 			Assert.IsNotNull(lv.ContainerFromItem(0));
 		}
 #endif
+
+#if HAS_UNO
+		[TestMethod]
+		[RunsOnUIThread]
+#if !HAS_INPUT_INJECTOR
+		[Ignore("InputInjector is only supported on skia")]
+#endif
+		public async Task When_UpdateLayout_In_DragDropping()
+		{
+			var SUT = new ListView
+			{
+				AllowDrop = true,
+				CanDragItems = true,
+				CanReorderItems = true
+			};
+
+			for (var i = 0; i < 3; i++)
+			{
+				SUT.Items.Add(new UpdateLayoutOnUnloadedControl
+				{
+					Content = new TextBlock
+					{
+						AllowDrop = true,
+						CanDrag = true,
+						Margin = new Thickness(50),
+						Text = i.ToString()
+					}
```

Comment:
would be better to explicitly set the height, than margin
```suggestion
					Content = new TextBlock
					{
						AllowDrop = true,
						CanDrag = true,
						Height = 100,
						Text = i.ToString()
					}
```